src/main.js

```diff
@@ -149,13 +150,19 @@ function update (job, logging) {
 
     github.clone(job, job.commit.sha, logging).then(() => {
       const name = (job.options ? job.options.prefix : 'DEV') + ' - ' + job.id + ' (git ' + job.commit.sha.substr(0, 8) + ')'
-      maven.setVersion(job, name, true).then(resolve, reject)
+      log(logging, `-> Building using: ${job.options.buildTool}`)
+      if (job.options.buildTool === 'maven') {
```

Comment:
I'd still just make this default to Maven

```suggestion
      if (!job.options.buildTool || job.options.buildTool === 'maven') {
```